apps/events-helsinki/src/domain/search/eventSearch/utils.tsx

```diff
@@ -211,24 +211,9 @@ export const getEventSearchVariables = ({
     MAPPED_EVENT_CATEGORIES
   );
 
-  const hasLocation = !isEmpty(places);
-
-  const getSearchParam = () => {
-    const hasText = !isEmpty(text);
-    if (hasText && hasLocation) {
-      // show helsinki events matching to text
-      return { localOngoingAnd: text };
-    } else if (hasText) {
-      // show internet and helsinki events matching to text
-      return { allOngoingAnd: text };
-    } else {
-      // show all internet and helsinki events
-      return { allOngoing: true };
-    }
-  };
```

Comment:
How the hasText/hasLocation combinations work after these changes would be good to test that they conform to product owner's expectations.